extensions/amp-fit-text/0.1/amp-fit-text.js

```diff
@@ -241,13 +242,30 @@ export function buildDom(document, element) {
   measurer.classList.add(MEASURER_CLASS);
 
   realChildNodes(element).forEach((node) => contentWrapper.appendChild(node));
-  measurer./*OK*/ innerHTML = contentWrapper./*OK*/ innerHTML;
+  mirrorNode(contentWrapper, measurer);
   element.appendChild(content);
   element.appendChild(measurer);
 
   return {content, contentWrapper, measurer};
 }
 
+/**
+ * Make a destination node a clone of the source.
+ *
+ * @param {!Node} source
+ * @param {!Node} dest
+ */
+function mirrorNode(source, dest) {
+  // First clear out the destination node.
+  removeChildren(dest);
+
+  // Then copy all the source's child nodes into destination node.
+  const {childNodes} = source;
+  for (let i = 0; i < childNodes.length; i++) {
```

Comment:
Nit: `childNodes.length` is a live-updating reference that exists in C++, which means you're calling from JS into C++ during every iteration (slow). Cache `length` locally, or use `nextSibling` iteration.

Reply:
I've never done a `nextSibling` iteration before....so I cached length